### PR TITLE
Show floor infomation with room on session details screen

### PR DESCRIPTION
**Pull request by hiraike32**
## Issue
- none

## Overview (Required)
- Added the floor number to the room information on the session details screen.

## Links
- [Figma](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=54902%3A40380&mode=dev)

## Screenshot
Before | After
:--: | :--:
Screenshot_20230818_163004 | Screenshot_20230818_164454 |

Japanese

| Arctic Fox | Bumblebee | Chipmunk | Dolphin | Electric eel |
| -- | -- | -- | -- | -- |
| Screenshot_20230818_165343 | Screenshot_20230818_165351 | Screenshot_20230818_165334 | Screenshot_20230818_165358 | Screenshot_20230818_165405 |

English
| Arctic Fox | Bumblebee | Chipmunk | Dolphin | Electric eel |
| -- | -- | -- | -- | -- |
| Screenshot_20230818_165456 | Screenshot_20230818_165502 | Screenshot_20230818_165449 | Screenshot_20230818_165508 | Screenshot_20230818_165514 |

### CODE REVIEW

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/TimetableRoom.kt
Comment (hiraike32) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/TimetableRoom.kt:21]: I exhausted the room so that we get a compilation error when we get more room.

File: core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/TimetableRoom.kt
Comment (hiraike32) [inline core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/TimetableRoom.kt:15]: In figma, it was written as "地下1階" in Japanese, so I changed the expression depending on the language.

スクリーンショット 2023-08-18 17 14 20


https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI?type=design&node-id=54462%3A36316&mode=dev

**Comment by takahirom** [comment]
Please wait until this is merged.
https://github.com/DroidKaigi/conference-app-2023/pull/648

**Comment by RyuNen344** [comment]
@hiraike32
Block is over!!
Could you resolve some conflicts?

**Comment by hiraike32** [comment]
I resolved the conflict, please review again.

**Review by takahirom — APPROVED** [review]
Thanks! It's very convenient!

**Pull request merged**
